Day-15_Classes/classes.js

```diff
@@ -0,0 +1,154 @@
+// Exercises Level 1
+
+// Create an Animal class. The class will have name, age, color, legs properties and create different methods
+class Animal {
+  constructor (name, age, color, legs) {
+    this.name = name
+    this.age = age
+    this.color = color
+    this.legs = legs  
+  }
+
+  get getName(){
+    return this.name
+  }
+
+  get getAge(){
+    return this.age
+  }
+
+  get getColor(){
+    return this.color
+  }
+
+  get getLegs(){
+    return this.legs
+  }
+
+  eat() {
+    console.log('Nyam nyam')
+  }
+
+  sleep() {
+    console.log(' ... zzzZZZ')
+  }
+
+  getFullInfo() {
+    const fullInfo = `This anymal's name is ${this.getName}. It is ${this.getAge} years old and of ${this.getColor} color. It has ${this.getLegs} legs`
+    return fullInfo
+  }
+}
+
+
+// Create a Dog and Cat child class from the Animal Class.
+class Dog extends Animal {
+  saySomething() {
```

Comment:
Since the Animal class accpets `name, age, color, legs` parameters in its constructor, Dog class needs accept those parameters too when instantiated and pass them to parent class through super() method called inside the constructor.

Reply:
Updated with constructor